chunkid.h

```diff
@@ -0,0 +1,26 @@
+#ifndef CHUNKID_H
+#define CHUNKID_H
+
+// ChunkID is the key used to identify entries in the Cache
+// Chunks are identified by their coordinates (CX,CZ) but a single key is needed to access a map like structure
+class ChunkID {
+ public:
+  ChunkID(int cx, int cz);
+  bool operator==(const ChunkID &) const;
+  friend unsigned int qHash(const ChunkID &);
```

Comment:
The previous declaration returned an `uint`; with bitwise operators that's the only safe way, some operations are undefined behavior on certain signed types. Are you still sure you need an `int`?

Reply:
I replaced `uint` with `unsigned int` which should be the very same at the end.
take a look into `qglobal.h` and you find this definition of uint: `typedef unsigned int uint;`
I just found it more elegant to not have any dependency to qt here in the header anymore.
But it's rather a subtle difference considering that the most things in minutor make use of Qt stuff ;-). So if you see an other reason why you would prefer `uint`over `unsigned int` I'll happy to change it back - no problem.

Reply:
Right, I'm blind, completely missed the `unsigned` there. Sorry.